repo-inventory/src/main/java/fr/lateb/presentation/rest/MenuResource.java

```diff
@@ -0,0 +1,52 @@
+package fr.lateb.presentation.rest;
+
+import fr.lateb.domain.entity.BeerEntity;
+import fr.lateb.domain.service.BeerService;
+import fr.lateb.util.BeerNotFoundError;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+@Tag(name = "Inventory", description = "Gestion des bières")
```

Comment:
```suggestion
@Tag(name = "Inventory", description = "Inventory managment")
```

Everything in English